orientation_covariance: [0.02, 0, 0, 0, 0.02, 0, 0, 0, 0.02]
angular_velocity_covariance: [0.01, 0, 0, 0, 0.01, 0, 0, 0, 0.01]
linear_acceleration_covariance: [0.05, 0, 0, 0, 0.05, 0, 0, 0, 0.05]
